src/components/Game/Game.js

```diff
@@ -173,9 +176,21 @@ export default class Game extends Component {
     });
   };
 
+  handleVimCommand = () => {
+    this.setState({
+      vimCommand: true,
+    });
+  };
+
+  handleVimCommandBuffer = (key) => {
+    this.setState({vimCommandBuffer: [...this.state.vimCommandBuffer, key]});
```

Comment:
can we should check if key is special key? e.g. backspace, shift & other modifiers, esc, who knows what. seems like esc works already but backspace & shift don't.